sys-power/suspend/files/suspend-1.0_p20200924-fix-loff_t-for-musl.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/load.c b/load.c
+index 6d1c6ed..adc0af5 100644
+--- a/load.c
++++ b/load.c
+@@ -8,6 +8,7 @@
+  * This file is released under the GPLv2.
+  */
+
++#define _GNU_SOURCE
+ #include "config.h"
+ #include <sys/types.h>
+ #include <sys/stat.h>
+diff --git a/suspend.c b/suspend.c
+index 479ce58..c82a85f 100644
+--- a/suspend.c
++++ b/suspend.c
+@@ -9,6 +9,7 @@
+  *
+  */
+
++#define _GNU_SOURCE
+ #include "config.h"
+ #include <sys/types.h>
+ #include <sys/stat.h>
+diff --git a/swsusp.h b/swsusp.h
+index 2aae184..8d22b92 100644
+--- a/swsusp.h
++++ b/swsusp.h
+@@ -9,6 +9,7 @@
+  *
+  */
+
++#define _GNU_SOURCE
+ #include <stdint.h>
```

Comment:
... as long as all files include config.h before anything else.

Reply:
It does work on the test musl VM.

Reply:
I was pointing out that one file doesn't do it so that should be fixed, otherwise you can get weird/broken behaviour.

Reply:
Oh, I see. I fixed that up.